src/arch/x86_64/kernel/kmain.c

```diff
@@ -167,6 +172,16 @@ void kmain(uint64_t addr)
   alloc_init();
   print_ok();
 
+#ifdef ENABLE_FRAMEBUFFER
+  vbe_video_init(&entry->common);
+  video_clear(MAKE_RGB(0, 128, 0));
+  video_console_attach();
```

Comment:
I am a bit unclear about why we need this, especially here. We have `console_mode_is_vbe()` to use the VBE framebuffer later in the boot sequence.

Reply:
because otherwise the double buffer won't print the text

Reply:
so I guess the console shouldn't use the vbe framebuffer anymore but a video buffer or something?

Reply:
basically, yeah. the double buffer would replace the framebuffer anyway.